sdk-docs/tutorials/flask.md

```diff
@@ -0,0 +1,491 @@
+# Write your first Kubernetes charm using the PaaS app charmer
+
+In this tutorial you will learn how to use the PaaS app charmer to convert your
+existing Flask application to a Kubernetes charm for Juju.
+
+What you’ll need:
+
+* A work station, e.g., a laptop, with amd64 architecture and which has
+  sufficient resources to launch a virtual machine with 4 CPUs, 8 GB RAM, and 30
+  GB disk space
+* Familiarity with Linux
+* Completed [Get started with Juju](https://juju.is/docs/juju/tutorial) or
+  familiarity with Juju
+
+What you’ll do:
+
+* [Study your application](#study-your-application)
+* [Set up your development environment automatically](#set-up-your-development-environment-automatically)
+
+1. Enable `juju deploy sample-flask`
+2. Enable `juju config sample-flask language=<language>`
+3. Enable `juju integrate sample-flask mysql-k8s`
+
+* Clean up: Destroy your test environment
+
+## Study your application
+
+In this tutorial we will be converting a small educational Flask application -
+`sample-flask` into a charm.
+
+You can check out the source code of the Flask application from the GitHub
```

Comment:
Oh so this is asking the user to download the code for the sample app being used in the tutorial, `checkout` is a meaningful term for `git` which essentially means download.

Reply:
I know, my point was about the "You can do x" construction. It's ambiguous in that it can be read as giving information or politely asking someone to do something. In this case we're just trying to give information. Hence my suggestion we find a non-ambiguous way to say just that.